### ISSUE TITLE
🐛 biome incorrectly reports 'aria-atomic' as an invalid ARIA attribute

### ISSUE THREAD

**Issue by teehemkay** [biomejs/biome#1167]
### Environment information

```block
CLI:
  Version:                      1.4.1
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           macos

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v18.18.2"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

1. run `biome lint <file-to-lint>`
2. <file-to-lint> contains a JSX tag with an aria-atomic="true"
3. biome reports an error: `The element contains invalid ARIA attribute(s)` and `ℹ aria-atomic is not a valid ARIA attribute.`

https://biomejs.dev/playground/?code=PABzAHAAYQBuAAoACQBpAGQAPQAiAGgAaQBuAHQALQBjAG8AdQBuAHQAcgB5ACIACgAJAHIAbwBsAGUAPQAiAGEAbABlAHIAdAAiAAoACQBhAHIAaQBhAC0AYQB0AG8AbQBpAGMAPQAiAHQAcgB1AGUAIgAKAAkAYwBsAGEAcwBzAE4AYQBtAGUAPQAiAGIAbABvAGMAawAgAGUAcgByAG8AcgAtAHQAZQB4AHQAIABwAHQALQAyACIACgA%2BAAoACQB7AGMAbwB1AG4AdAByAHkARQByAHIAbwByAH0ACgA8AC8AcwBwAGEAbgA%2BADsA

### Expected result

biome should not report an error since `aria-atomic` is a valid ARIA attribute.
https://w3c.github.io/aria/#aria-atomic

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
This should be relatively easy to fix. We have `aria-atomic` in our database, but we probably missed some metadata to track it.
